cognite/client/_api/events.py

```diff
@@ -198,7 +200,12 @@ def list(
             type=type,
             subtype=subtype,
         ).dump(camel_case=True)
-        return self._list(method="POST", limit=limit, filter=filter)
+        if partitions and (limit is not None and limit != -1 and limit != float("inf")):
```

Comment:
Duplicate code with the one for assets

Reply:
duplication of this type is pretty typical, and we don't pass limit to the function, so hard to check it there.

Reply:
on second thought, done